src/index.html

```diff
@@ -10,8 +10,219 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="field">
+
+          <label class="label">
+          Surname:
+            <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
            <input
              type="text"
              name="Surname"
              autocomplete="off"
            >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) fix code style all over the page